test/mason/MasonTestHelpers.chpl

```diff
@@ -1,9 +1,13 @@
 use Spawn;
 
-proc checkExitStatus(cmd, exitStatus) {
+proc checkExitStatus(cmd: [] string, exitStatus: int) {
   var p = spawn(cmd, stdout=PIPE);
   p.wait();
   if p.exitCode == exitStatus {
     writeln("Got exit status %i as expected".format(exitStatus));
+  } else {
+    writeln("Error: %s returned %i when test expected %i".format(" ".join(cmd),
```

Comment:
I see it was this way before, but both this and the above call are more straightforwardly written as `writef("bla\n", args)`